example/renovate-config.js

```diff
@@ -22,8 +22,8 @@ module.exports = {
         'major',
         'lockFileMaintenance',
       ],
-      masterIssueApproval: false,
-      stabilityDays: 0,
```

Comment:
This one is still named `stabilityDays` and incorrectly changed to `matchUpdateTypes`. The rest looks good to me

Reply:
Oh, sorry about that. I am not sure what happend in that update. I have now updated the correct option.